pkg/installer/installation/prerequisites.go

```diff
@@ -26,7 +26,174 @@ import (
 	"github.com/kubermatic/kubeone/pkg/util"
 )
 
-const dockerVersion = "18.09.2"
+const (
+	dockerVersion = "18.09.6"
+)
+
+const (
```

Comment:
Why do you repeat `const` two times?

Reply:
it's two blocks of consts, 1 for "constants" and 1 for "scripts"

Reply:
Usually, that doesn't matter at all, but I'm not going to block on this.